### PR TITLE
Create 2024-03-15.md

### PR DESCRIPTION
**Pull request by camronmartinez**

### CODE REVIEW

File: content/verification/veteran-verification/release-notes/2024-03-15.md
Comment (melliroo) [inline content/verification/veteran-verification/release-notes/2024-03-15.md:1]: A few questions: 
 - Change applies only to v2? 
 -  applies to each of the endpoints? 
 - what's a 'rating profile'?

Comment (camronmartinez) [reply content/verification/veteran-verification/release-notes/2024-03-15.md:1]: Hi, sorry on the delay:

- Yes, the changes only apply to v2.
- It applies to each of the disability rating endpoints. 
image
- After speaking with Lorri, it might make more sense to say "current profile" instead of "current rating profile". A rating profile is a view of all the vet's disability ratings based on their status. There is the concept of a "current" rating profile that is considered "active" which can also include a disability rating with an end date in the past.....you would think that means expired.  But nope, because it's in the current rating profile.  (I think in the mega VA system where va assessors enter, evaluate and process disability claims, they must manually SET something as expired.  Then it would have an end date PLUS an expired status and I think it gets thrown out of the current rating profile.) 

So, in short, of the total existing profiles for that veteran, we're specifying the current "active" profile.

Comment (melliroo) [reply content/verification/veteran-verification/release-notes/2024-03-15.md:1]: Thanks for the details! Does the "current profile"  / "current rating profile" live in a VA system (for example, in VA Profile, VBMS, or BGS )?  I'm wondering if that concept is specific to the VA system. If it is, wondering if it's something our current and future consumers (who are developers) are aware of or need to be aware of.

Comment (camronmartinez) [reply content/verification/veteran-verification/release-notes/2024-03-15.md:1]: I'm not completely sure but this is what I understand: It exists in a large system within VA and pieces get sent to BGS, VA Profile to be used as source data.  I believe the mega system is referred to most often as VA Corp? I'd have to look into that though. I think it could be a good idea to better define what we mean by "profile" and give that definition to consumers. Could help to give context to where the info is coming from?

Comment (melliroo) [reply content/verification/veteran-verification/release-notes/2024-03-15.md:1]: Ok, thanks!  I'm going to make some suggestions for edits

**Review by melliroo — COMMENTED** [review]
Please let me know if I've captured this accurately, but for consideration: 

We modified the API response for the disability ratings endpoints for v2 of the Veteran Service History and Eligibility API. 
* We no longer return disability ratings with an end date in the past. 
* A response with no disability rating now contains a blank array in the data fields instead of [Null].

To learn more, review the [Veteran Service History and Eligibility v2 documentation](https://developer.va.gov/explore/api/veteran-service-history-and-eligibility/docs?version=current).

**Comment by camronmartinez** [comment]
Suggested changes look good. Committed a new version that reflects them.

**Review by melliroo — APPROVED** [review]
